### PR TITLE
nit: tweak red highlight

### PR DESCRIPTION
**Pull request by UnloadingGnat**
## Motivation

#53 

Red selected highlight doesn't match the rest of the UI.

## Solution

Changed to #4285F4 a blue color, also the google blue color!

image

## Checklist

- [X] Tested in Chrome
- [x] Tested in Safari

---

## PR-Codex overview
This PR updates the box-shadow styling in `src/index.css` and replaces the red color with `#E73324`.

### Detailed summary
- Replaced red color with `#E73324` in `box-shadow` styling in `src/index.css`.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by UnloadingGnat** [comment]
Hey @transmissions11 long time, since 2020 summer code jam. Been super interested in ChatGPT and LLMs recently and I think this is an awesome project would love to contribute!

**Comment by francoabaroa** [comment]
@UnloadingGnat sweet! For me, blue might be confusing as the highlight color especially because 2 different blue's are already used twice: #[5F8AF7](https://convertingcolors.com/hex-color-5F8AF7.html) and #[C5E2F6](https://convertingcolors.com/hex-color-C5E2F6.html)


blue

Maybe you can try using an Amber yellow #[FFBF00](https://convertingcolors.com/hex-color-FFBF00.html), it goes well with the existing palette (or the websafe version #[FFCC00](https://convertingcolors.com/hex-color-FFCC00.html))


palette

flux

flux2

If you don't like the yellow, you can also use #[DE6B48](https://convertingcolors.com/hex-color-DE6B48.html) (or the websafe version #[CC6633](https://convertingcolors.com/hex-color-CC6633.html))

palette2

#DE6B48

Or #[E26D5C](https://convertingcolors.com/hex-color-E26D5C.html) (websafe version #[CC6666](https://convertingcolors.com/hex-color-CC6666.html))

palette3

#E26D5C

Feel free to ignore, just a suggestion :)

CC: @transmissions11

File: src/index.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .selected {
-  box-shadow: 0px 0px 0px 2px red, 0px 0px 30px 2px red !important;
+  box-shadow: 0 0 0 2px #4285F4 !important;
```
Comment (transmissions11) [inline src/index.css:2]:
```suggestion
  box-shadow: 0 0 0 2px #E73324 !important;
```

**Comment by transmissions11** [comment]
thanks @francoabaroa and @UnloadingGnat for your input here — ended up going with something similar to orangy-red @francoabaroa selected

**Pull request merged**